erica/erica_legacy/elster_xml/elster_xml_generator.py

```diff
@@ -267,7 +267,7 @@ def _add_vast_request_xml_nutzdaten(xml_top, user_data):
     spez_recht_antrag_xml = SubElement(nutzdaten_xml, 'SpezRechtAntrag')
     spez_recht_antrag_xml.set('version', '3')
     SubElement(spez_recht_antrag_xml, 'DateninhaberIdNr').text = user_data['idnr']
-    SubElement(spez_recht_antrag_xml, 'DateninhaberGeburtstag').text = user_data['dob']
+    SubElement(spez_recht_antrag_xml, 'DateninhaberGeburtstag').text = user_data['date_of_birth']
```

Comment:
we need to update the webapp request to erica also if you change the data name or?

Reply:
I was afraid of that at first, too. However, we not need to do that. That's why I added the alias in the erica_input.py. I commented the relevant lines below. Maybe have a look there?